media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="http://flac-tools.sourceforge.net/"
+COPYRIGHT="2003-2017 Jake Angerman"
```

Comment:
Jake Angerman is one of the original authors. Maybe we should add the other one, and the current maintainer too.
The only problem is that the AUTHORS file does not provide any copyright years.

So I don't know if
```
COPYRIGHT="2003-2012 Jake Angerman, Dan Johnson, Hans Oesterholt"
```
would be acceptable.